### PR TITLE
Fix syntax errors

### PR DESCRIPTION
**Pull request by Ladsgroup**
and some minor pep8

### CODE REVIEW

File: diffengine/stores/mongo.py [hunk 1]
```diff
@@ -11,18 +11,18 @@
...
     def __init__(self, db):
         self.db = db
         self.revisions = Revisions(self)
         self.engine_status = EngineStatus(self)
         self.processor_status = ProcessorStatus(self)
-    
+
     @classmethod
-    def from_params(cls, *args, db_name, **kwargs):
+    def from_params(cls, db_name, *args, **kwargs):
```
Comment (halfak) [inline diffengine/stores/mongo.py:22]: In python3, this is a way to make keyword arguments required. 

```
>>> def foo(*args, bar, **kwargs):
...     pass
... 
>>> foo("test")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() missing 1 required keyword-only argument: 'bar'
>>> foo(bar="test")
```

File: doc/notes_on_celery.py [hunk 2]
```diff
@@ -60,31 +60,31 @@ def process_revisions(page_id, page_revs):
 @task_postrun.connect()
 def task_postrun(*args, **kwargs):
     # always, no matter exceptions or not, remove database sessions
-    
-    
+    return
```
Comment (halfak) [inline doc/notes_on_celery.py:63]: This is just documentation and it is intended to be interpreted as psuedo-code.  See "listener = " below.

Comment (Ladsgroup) [inline doc/notes_on_celery.py:63]: you're right, I will change this.

Best

On Mon, Feb 9, 2015 at 12:00 AM, Aaron Halfaker notifications@github.com
wrote:

> In doc/notes_on_celery.py
> https://github.com/halfak/Difference-Engine/pull/1#discussion_r24301981:
> 
> > @@ -60,31 +60,31 @@ def process_revisions(page_id, page_revs):
> >  @task_postrun.connect()
> >  def task_postrun(_args, *_kwargs):
> > 
> > ##      # always, no matter exceptions or not, remove database sessions
> > 
> > ## 
> > -    return
> 
> This is just documentation and it is intended to be interpreted as
> psuedo-code. See "listener = " below.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/halfak/Difference-Engine/pull/1/files#r24301981.

## 

Amir

**Comment by Ladsgroup** [comment]
Very long time ago, I was stupid :D

**Pull request closed without merging**